### DISCUSSION TITLE
mspider/intro/faq

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2780] [Announcements]
# mspider/intro/faq

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/faq.html

**Comment by samzong** [comment]
常见问题 5 ，目前 DSM 已有方案迁移到 DCE5 之中，此次答案描述需要更新，具体方案参考最佳实践。

#2781
